chain/consensus/hierarchical/actors/sca/sca_actor.go

```
@@ -424,7 +429,8 @@ func (a SubnetCoordActor) Fund(rt runtime.Runtime, params *SubnetIDParam) *abi.E
 	// Increment stake locked for subnet.
 	var st SCAState
 	rt.StateTransaction(&st, func() {
-		msg := fundMsg(rt, address.SubnetID(params.ID), secpAddr, value)
+		sn, _ := address.SubnetIDFromString(params.ID)
```

Why error is ignored here?